include/psen_scan/scanner.h

```diff
@@ -34,6 +35,8 @@ class vScanner
 };
 // LCOV_EXCL_STOP
 
+constexpr std::chrono::seconds FETCH_FRAMES_TIMEOUT{ std::chrono::seconds(1) };
```

Comment:
The naming is a little confusing since in practice the timeout does not stay constant.

Reply:
Done.